src/modules/vcsh.rs

```diff
@@ -0,0 +1,76 @@
+use super::{Context, Module};
+
+use crate::config::RootModuleConfig;
+use crate::configs::vcsh::VcshConfig;
+use crate::formatter::StringFormatter;
+
+/// Creates a module that displays VCSH repository currently in use
+///
+/// Will display the name of the current VCSH repository if one is active.
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let repo = context
+        .get_env("VCSH_REPO_NAME")
+        .unwrap_or_else(|| "".into());
```

Comment:
Just one small nitpick.
```suggestion
        .unwrap_or_default();
```